### DISCUSSION TITLE
Show exact number of bytes?

### DISCUSSION THREAD

**Discussion by gustafbstrom** [bootandy/dust#374] [General]
Using `-H`, we get seemingly get the default behavior, i.e., a summary of the top level directories and their sizes shown in the most appropriate order of magnitude ("1.1G", for example). I'm in a scenario where I want to see the exact number of bytes, but I can't see a flag for that. Is there a technical reason for this (i.e., dust is extra fast due to approximations) or is it just a matter of no use until now (unless I'm completely missing something, of course).

Last, thanks everyone involved for a beautiful tool. Cheers.

**Comment by bootandy** [comment]
There's not yet. There's a PR for that: https://github.com/bootandy/dust/pull/372

It'll be in the next release.

**Reply by gustafbstrom** [reply]
Cool. Best wishes.

**Comment by bootandy** [comment]
and -H is for 'si' so a KiB not a KB ie 1000 not 1024.
